docs/en/revocation-lists.rst

```diff
@@ -57,7 +57,7 @@ Functional Requirements
 - revoke a Digital Credential when the following circumstances occur:
 
   - the Digital Credential requires to be updated, whenever one or more attributes are changed; in this case the User will request a new issuance for that Digital Credential;
-  - the Wallet Instance that holds the Digital Credential was issued is revoked;
+  - the Holder requests such revocation due to the loss or replacement of cryptographic key material to which the issued Digital Credential is bound. In such case, the Holder should request a revocation using a web service provisioned by the Wallet Provider and an authentication method with a High Level of Assurance (e.g., CIE id level 3);
```

Comment:
`replacement of cryptographic key material to which the issued Digital Credential is bound` sounds misleading, if the digital credential has a binded cryptographic material this is not supposed to be replaced